catalyst_voices/packages/internal/catalyst_voices_view_models/lib/src/campaign/campaign_info.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:catalyst_voices_view_models/src/campaign/campaign_stage.dart';
+import 'package:equatable/equatable.dart';
+
+sealed class CampaignInfo extends Equatable {
```

Comment:
Does it really make sense to have sealed class instead of one additional enum?

Reply:
Will convert to enum

Reply:
Removed sealed class, added simple class with enum